config/initializers/countries.rb

```diff
@@ -1,13 +1,25 @@
 ISO3166::Data.register(
-  alpha2: 'KO',
+  alpha2: 'XK',
```

Comment:
I know that this is the official code of Kosovo (and we had it like that until recently), but it was changed so as to have a more proper sorting of nationalities.